src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -12,17 +12,57 @@ using namespace AppInstaller::Utility;
 using namespace AppInstaller::Utility::literals;
 
 namespace {
-    void ShowMultiLineField(Execution::OutputStream& outputStream, AppInstaller::StringResource::StringId label, std::string& value)
+
+    template <typename String>
+    void ShowSingleLineField(Execution::OutputStream outputStream, const AppInstaller::StringResource::StringId& label, const String& value, bool indent = false)

```

Comment:
nit: StringId is actually a string_view, they'd better be passed by value.

Same for the MultiLine and multivalue one